src/Template/Aefis/xml/e2b.ctp

```diff
@@ -75,9 +75,9 @@
         </reportduplicate>
         <?php $arr = preg_split("/[\s]+/", $aefi['reporter_name']); ?>
         <primarysource>
-            <reportergivename><?php if (isset($arr[0])) echo $arr[0]; ?></reportergivename>
-            <reporterfamilyname><?php if (isset($arr[1])) echo $arr[1].' '; if (isset($arr[2])) echo $arr[2];  ?></reporterfamilyname>
-            <reporterorganization><?php echo $aefi['name_of_vaccination_center']; ?></reporterorganization>
+            <reportergivename><?php if (isset($arr[0])) echo preg_replace('/[^A-Za-z0-9\s+]/', '',$arr[0]); ?></reportergivename>
+            <reporterfamilyname><?php if (isset($arr[1])) echo preg_replace('/[^A-Za-z0-9\s+]/', '',$arr[1]).' '; if (isset($arr[2])) echo preg_replace('/[^A-Za-z0-9\s+]/', '',$arr[2]);  ?></reporterfamilyname>
+            <reporterorganization><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefi['name_of_vaccination_center']); ?></reporterorganization>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate